### DISCUSSION TITLE
CPU usage increases after SRT Playback ends

### DISCUSSION THREAD

**Discussion by Seok231** [HaishinKit/HaishinKit.swift#1421] [Q&A]
Hello, first of all, I'm using a translator, so the sentence might be a little awkward.

I was using SRT Publish, PlayBack on Example iOS and found something weird.

First of all, there is no problem during SRT transmission or Playback. However, after the SRT Playback was terminated, it was confirmed that CPU usage increased.

SRT Publish( CPU 45~50%)

SRT PlayBack(CPU 25~35%)

stream.close()
connection.close()
CPU usage has increased to 100-110% since PlayBack ended.

The code used was the project's Example iOS.

The SRT server I used is "https://github.com/Edward-Wu/srt-live-server".

Has anyone experienced the same symptoms?

netStreamSwitcher.open(.playback)
![image](https://github.com/shogo4405/HaishinKit.swift/assets/97385742/4154f46a-7758-4e44-ac09-f6de27a432ae)

netStreamSwitcher.close()
<img width="1743" alt="image" src="https://github.com/shogo4405/HaishinKit.swift/assets/97385742/834c6609-0cc3-453a-84b8-5d147b562ab8">

**Comment by shogo4405** [comment]
I can reproduce it locally, so I think it's just a bug. Closes and moving to issue https://github.com/shogo4405/HaishinKit.swift/issues/1426
